pkg/s3-proxy/server/tls.go

```diff
@@ -0,0 +1,228 @@
+package server
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"errors"
```

Comment:
Can you replace that with "github.com/pkg/errors"? That will allow having stack traces.

Reply:
Yes, however... (see next comment)